events.json

```diff
@@ -34,5 +34,7 @@
   "API_LAYER_UPDATED" : "v1/API_LAYER_UPDATED",
   "API_LAYER_UPDATE_ERROR" : "v1/API_LAYER_UPDATE_ERROR",
   "SIO_MISSING_PROJECTION_ERROR" : "v1/SIO_MISSING_PROJECTION_ERROR",
-  "SIO_MISMATCHED_GEOMETERY_TYPE_ERROR" : "v1/SIO_MISMATCHED_GEOMETERY_TYPE_ERROR"
+  "SIO_MISMATCHED_GEOMETERY_TYPE_ERROR" : "v1/SIO_MISMATCHED_GEOMETERY_TYPE_ERROR",
```

Comment:
@mtCarto  is this a generic error across all files?

Reply:
This should be a shapefile error. I haven't seen it in the other data formats I've tested so far.

This error is due to Polygons being present in a MultiPolygon type (and vice versa). as documented in [SVC-670](https://issues.boundlessgeo.com:8443/browse/SVC-670).  If we improve how that's handled we won't need this error.

Reply:
Then we should add the new standard format of `SIO_SHP_` prefix